services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -302,6 +303,8 @@
  */
 public class VeniceParentHelixAdmin implements Admin {
   private static final long SLEEP_INTERVAL_FOR_DATA_CONSUMPTION_IN_MS = 1000;
+  private static final long SLEEP_INTERVAL_ROLL_FORWARD_CHECK_IN_MS = 1000;
```

Comment:
Can we reduce the sleep time? generally rollforward should be very fast. If it fails generally it means will not succeed in the next time. Only thing I think it will fail transiently is network issues. Also @misyel deferred swap service cannot tolerator this long delay in the parent controller for failed roll-forward as it will affect too many stores.